README.md

```diff
@@ -1,2 +1,89 @@
 # GOB
-Generieke Ontsluiting Basisgegevens
+
+Generic Disclosure of Key Registrations
+
+_Een Nederlandse (beperkt onderhouden) versie van dit document is beschikbaar in [README.nl](README.nl.md)_
+
+## Background
+
+Key registrations provide for basic data; the raw material for a large number of municipal processes.
+Without this data, the processes can not be executed.
+
+Basic data are recorded in key registers (prescribed by law) and core registrations (prescribed by the Municipal Executive).
+
+Some of these data are taken from their own municipal facilities (source system) and partly from national facilities.
+The problem is that all these sources do not use one and the same data / message standard.
+In-house customers make use of different systems in which the basic data are shown or processed.
+
+The seven key registers are:
+1. persons (BRP)
+2. addresses and buildings (BAG)
+3. value of immovable property (BR WOZ)
+4. small-scale topography (BRT)
+5. large-scale topography (BGT)
+6. Commercial register (HR)
+7. Land Registry (BRK).
+
+The eight core registrations are:
+1. the municipal restriction registration (Wkpb)
+2. the current height file in the Netherlands
+3. areas
+4. NAP
+5. aerial photographs
+6. measuring bolts
+7. monuments
+8. panorama images.
+
+Three systems are in use for retrieving data from the source or the national facility, the management, the translation and the distribution of basic data:
+- DIVA for real estate data
+- Makelaar suite for personal details
+- Handelsregister and Neuron Communicator for BAG and WOZ data
+
+Not all systems can handle the different deliveries.
+A 'translation' is therefore necessary.
+Within GOB, basic data from the various source registrations are stored, processed and distributed in a format as defined in
```

Comment:
add: `the standards set by the City of Amsterdam and the association of Dutch municipalities`

Reply:
Done